packages/sources/cryptoapis-v2/src/config.ts

```diff
@@ -2,11 +2,35 @@ import { Requester } from '@chainlink/ea-bootstrap'
 import { Config } from '@chainlink/types'
 
 export const DEFAULT_ENDPOINT = 'price'
-export const NAME = "CRYPTOAPIS_V2"
+export const NAME = 'CRYPTOAPIS_V2'
 
 export const makeConfig = (prefix?: string): Config => {
   const config = Requester.getDefaultConfig(prefix, true)
   config.api.headers['X-API-Key'] = config.apiKey
   config.api.baseURL = 'https://rest.cryptoapis.io'
   return config
 }
+
+export const BLOCKCHAIN_NAME_MAP: { [key: string]: string } = {
```

Comment:
```suggestion
export const BLOCKCHAIN_NAME_BY_TICKER = {
```

Can we leave it as an object literal? It would make it more strict.
Also, we know its a mapping by its type definition, instead we can make the naming be more semantic as suggested